dlt/cli/config_toml_writer.py

```diff
@@ -36,10 +36,12 @@ def generate_typed_example(name: str, hint: AnyType) -> Any:
             return True
         if sc_type == "json":
             if is_subclass(inner_hint, C_Sequence):
-                return ["a", "b", "c"]
+                return ["customers", "orders", "items"]
```

Comment:
This seems very specific to the rest_api (same below). This might totally mess up instantiation of verified sources, have you thought about that?

Reply:
I checked against all verified sources and only chess is using this (atm)
to instantiate the secret_dict

Reply:
which, to be fair, looks weird:

```toml
[sources.chess.secret_dict] # fill this in!
base_url = "https://jaffle-shop.scalevector.ai/api/v1" # config object example

[sources.chess.secret_dict.paginator] # nested config object example
type = "header_link"
```

Reply:
@djudjuu this code applies to all configs, also configs for sources that user have written themselves, so we do not know when they will be applied. If you have discussed something else with @rudolfix then please disregard my comment, but I would really not put rest_api defaults into generic code. I thought we would use templates in dlt+, and I think we can probably skip the config toml for sources in the core repo, no? We use those templates there, or is there a reason to keep them?

Reply:
jup, this is disregarded already, I guess I could have deleted the comment. we will just not write values in OSS for complex types
(and do templates in dlt+)